WordPress/src/androidTest/java/org/wordpress/android/e2e/pages/StatsPage.kt

```diff
@@ -66,6 +67,12 @@ class StatsPage {
     }
 
     fun assertVisits(visitsData: StatsVisitsData): StatsPage {
+        // Skip this check for JP because of the bug with Stats card load.
+        // See https://github.com/wordpress-mobile/WordPress-Android/issues/18065
+        if (BuildConfig.IS_JETPACK_APP) {
+            return this
+        }
```

Comment:
What do you think about using the condition the other way around, like this:
```
        // Skip this check for JP because of the bug with Stats card load.
        // See https://github.com/wordpress-mobile/WordPress-Android/issues/18065
        if (!BuildConfig.IS_JETPACK_APP) {
            val cardStructure = Espresso.onView(
                Matchers.allOf(
                    ViewMatchers.isDescendantOfA(visibleCoordinatorLayout),
                    ViewMatchers.withId(R.id.stats_block_list),
                    ViewMatchers.hasDescendant(
                        Matchers.allOf(
                            ViewMatchers.withText("Views"),
                            ViewMatchers.hasSibling(ViewMatchers.withText(visitsData.views))
                        )
                    ),
                    ViewMatchers.hasDescendant(
                        Matchers.allOf(
                            ViewMatchers.withText("Visitors"),
                            ViewMatchers.hasSibling(ViewMatchers.withText(visitsData.visitors))
                        )
                    ),
                    ViewMatchers.hasDescendant(
                        Matchers.allOf(
                            ViewMatchers.withText("Likes"),
                            ViewMatchers.hasSibling(ViewMatchers.withText(visitsData.likes))
                        )
                    ),
                    ViewMatchers.hasDescendant(
                        Matchers.allOf(
                            ViewMatchers.withText("Comments"),
                            ViewMatchers.hasSibling(ViewMatchers.withText(visitsData.comments))
                        )
                    )
                )
            )
            cardStructure.check(ViewAssertions.matches(ViewMatchers.isCompletelyDisplayed()))
        }
        return this
```
This style of code seems to be used more consistently across the repo, and using this way, we could use the same return value for both cases.

Reply:
I like this much better @jostnes, thank you!